packages/vue/src/components/atoms/SfInput/SfInput.stories.js

```diff
@@ -127,12 +164,94 @@ storiesOf("Atoms|Input", module)
         :name="name"
         :valid="valid"
         :error-message="errorMessage"
-        :required="required"
+        :required="required"        
```

Comment:
ok, last comment about it, maybe some prettier changes :P

Reply:
yes I think so.